ci/run-tests.ps1

```
@@ -32,8 +33,6 @@ function Parse-UnrealOptions {
     return $options_result
 }
 
-. "$PSScriptRoot\common.ps1"
```

This caused some nasty silent failures